### ISSUE TITLE
Ability to use Azure Comment Reporter without publishing pipeline artifacts

### ISSUE THREAD

**Issue by ashokm** [oxsecurity/megalinter#2087]
TLDR - Broken links on Pull Request comments and Email notifications if users decide not to upload MegaLinter reports as part of their build pipeline

-----

When using [Azure Pipelines](https://github.com/oxsecurity/megalinter/tree/62fc5d3659aee4640a3b40468bf4409001968f15#azure-pipelines) and the [Azure Comment Reporter](https://github.com/oxsecurity/megalinter/blob/62fc5d3659aee4640a3b40468bf4409001968f15/docs/reporters/AzureCommentReporter.md), there are use cases where users would choose **not** to upload MegaLinter reports as part of their build pipeline, but only have the reports in the build output and Pull Request comments.

In these cases where the `PublishPipelineArtifact@1` task is not used, I would expect that the links in the Azure Comment Reporter and Email notifications link back to the buildId page and not to the artifacts page (&view=artifacts&pathAsName=false&type=publishedArtifacts).

If there is no automated approach in being able to determine whether artifacts are published, then perhaps a new environment variable is needed (to allow users to enable/disable) to allow the links to go back to the buildId page or the artifacts page accordingly.

**Comment by nvuillam** [comment]
@ashokm PR on the way :)

You 'll have to add `AZURE_COMMENT_REPORTER_LINKS_TYPE: build` in .mega-linter.yml :)
